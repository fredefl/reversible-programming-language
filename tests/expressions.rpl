read test;
test += 5;
hast += test + 3;
hast += test * 5;
hast += test / 20;
hast += test - 5;
read goodstuff;
hast -= goodstuff;
test += hast;
print test;
print hast;
print goodstuff;