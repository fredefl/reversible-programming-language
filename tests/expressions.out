38
28
40